pkg/kubelet/kubelet_node_status.go

```diff
@@ -319,16 +321,57 @@ func (kl *Kubelet) syncNodeStatus() {
 	}
 }
 
+// A notifier to make sure logs express backoff state when the kubelet is in trouble.
+var backoffNotifierRunning = false
+
+func (kl *Kubelet) backoffNotifier() {
```

Comment:
If I read your code correctly: The duration (seconds) of Kubelet retry its status next time in the following order;
01,02,04,08,16,32,64,128,356... 

I am not sure about the API server's startup latency today, especially on HA master case. But from above the time duration list, Kubelet tries to update the status ~7 times in the first minute, then will retry at ~2mins, then another try after 5mins (356s), at which all pods on the node should be evicted. 

Here is my concern based on your current algorithms if I am right above: If an API server's readiness / availableness latency is more than 1 mins, any restarting API server might cause node's status becoming Unknown, and even pods were evicted by reaching 5mins max timeout.

cc/ @k8s-mirror-api-machinery-api-reviews for API server's startup latency, @gmarek @wojtek-t on the node controller